db/README.md

```diff
@@ -33,6 +33,8 @@ The script runs in a single transaction.
 The following checklist summarizes what needs to be written to modify the database.
 
 * [ ] new version file in `db/versions` that updates all impacted stored functions
+  * All DB functions must continue to function for two major Taskcluster versions after they are used.  See [db/fns.md](db/fns.md) to figure out which existing functions you must re-implement.
+  * For any DB functions that are being deprecated in this version, add `deprecated: until vNN` in the re-implementation of that function, giving a major version two greater than the current version.
```

Comment:
`deprecated until` sounds like it will become undeprecated at that version. maybe there's a better word here

Reply:
Yeah, I don't like it either.  What about
```yaml
deprecated: v33
```
and that translates into "(support guaranteed until v35)" by adding 2?

Reply:
I like `deprecated by:  vXX` or `deprecated as of: vXX`

Reply:
Thanks! I think I'll go with
```yaml
deprecated-by: vXX
```